### PR TITLE
Feat/organizations and teams

### PR DESCRIPTION
**Pull request by joseglego**
# Cambios
- Mover el selector de _Organization_ del header al header de la Matriz
- Poder filtrar por _Team_  la matriz para mostrar la info

## Screenshots
- Antes:
00-antes

- Ahora:
01-sin-filtro
02-filtrado

### CODE REVIEW

File: app/assets/stylesheets/app/select.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  &__btn {
+    background-color: $bg-primary-color;
+    text-align: left;
+    width: 200px;
+    padding: 10px 25px 10px 15px;
+    position: relative;
+    display: inline-block;
+    height: 40px;
+
+    img, .octicon {
```
Comment (monkeyci) [inline app/assets/stylesheets/app/select.scss]: Expected newline after ","   selector-list-comma-newline-after

File: app/assets/stylesheets/app/select.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
+.select {
+  margin-left: 10px;
+
+  &:nth-child(1) {
+    margin-left: 0px;
```
Comment (monkeyci) [inline app/assets/stylesheets/app/select.scss]: Unexpected unit              length-zero-no-unit

File: app/controllers/organizations_controller.rb [hunk 3]
```diff
@@ -43,12 +43,16 @@ def load_organization
...
     end
-
     if @github_organization.nil?
       redirect_to '/organizations'
     else
       @organization = Organization.find_by(gh_id: @github_organization[:id])
       @has_dashboard = !@organization.nil?
+      if @has_dashboard
+        @teams = github_teams
+        @team = @teams.find { |team| team[:slug] == permitted_params[:team] } if permitted_params[:team]
```
Comment (monkeyci) [inline app/controllers/organizations_controller.rb]: Line is too long. [104/100]

File: app/views/organizations/_dashboard_table.html.erb [hunk 4]
```diff
@@ -1,51 +1,85 @@
 <div class="card-extended">
   <div class="card-extended__header">
-    <span class="card-extended__title">
-      <%= t('messages.dashboard.title', org: @organization.login.capitalize) %>
-    </span>
+    <% unless @github_session.organizations.empty? %>
+      <dropdown class="select">
+        <div class="select__btn" slot="btn">
+          <img src="<%= @github_organization[:avatar_url] %>" style="width:20px;">
```
Comment (oaestay) [inline app/views/organizations/_dashboard_table.html.erb]: 🤨

**Pull request merged**